lib/ui/compositing.dart

```diff
@@ -272,10 +272,10 @@ class SceneHost extends NativeFieldWrapperClass2 {
   /// to the child.
   ///
   /// The scene host takes ownership of the provided export token handle.
-  SceneHost(int export_token_handle) {
+  SceneHost(dynamic export_token_handle) {
```

Comment:
I want to accept Handle from dart:fidl.internal. This is called by //apps/mozart/lib/flutter/child_view.dart.

I made this dynamic since AIUI this code is compiled on all Flutter platforms so can't depend on Fuchsia-specific Dart SDK extensions. Is there a better way?